tests/e2e/util.go

```diff
@@ -3152,3 +3174,24 @@ func statefulSetFromManifest(fileName string, ss *appsv1.StatefulSet) (*appsv1.S
 
 	return ss, nil
 }
+
+// collectPodLogs collects logs from all the pods in the namespace
+func collectPodLogs(ctx context.Context, client clientset.Interface, namespace string) {
+	pods, err := client.CoreV1().Pods(namespace).List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	for _, pod := range pods.Items {
+		framework.Logf("Collectiong log from pod %v", pod.Name)
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		filename := pod.Name + curtimestring + val
+
+		//Collect Pod logs
+		cmd := []string{"logs", "--namespace=" + namespace, pod.Name, "--all-containers > ./" + filename}
```

Comment:
Dev insists that we copy logs from `/var/log/containers/<csi-pod-folder>` for csi pod logs or collect it container by container in separate files

Reply:
What difference does it make to get the logs from kubectl and reading from the file - they should be both same at any given time?

As observed on the WCP and GC setups, the csi pods won't be residing on the master VM where we do interact using kubectl command 
We need to identify the control-plane node where the csi pods are running and login to it and fetch the logs out of them to the worker machine.

Please feel free to create a task and assign it to me if this needs to be taken care.

AFAIK, both the logs should be similar, and fetching it from either way should be fine. Resolving the comment for now.